var/spack/repos/builtin/packages/pastix/package.py

```diff
@@ -12,12 +12,12 @@ class Pastix(CMakePackage, CudaPackage):
     based on direct methods"""
 
     homepage = "https://gitlab.inria.fr/solverstack/pastix/blob/master/README.md"
-    url = "https://gitlab.inria.fr/solverstack/pastix/uploads/baa57033d98378e0f3affbf45900fb6e/pastix-6.2.1.tar.gz"
+    url = "https://gitlab.inria.fr/solverstack/pastix/uploads/d579e932b415103974ced5d216481332/pastix-6.2.2.tar.gz"
```

Comment:
To answer the comment from @alalazo, if we manage multiple versions, the url should be constructed with:

"https://files.inria.fr/pastix/releases/v6/pastix-" + version + ".tar.gz" instead of using the gitlab one.

Reply:
That would be perfect, but you can also stick a specific `url=...` argument in the `version` directive.